bin/cliain/src/commands.rs

```diff
@@ -0,0 +1,238 @@
+
+    /// Change the validator set for the session after the next
+    ChangeValidators {
+        /// The new validators
+        #[clap(long, value_delimiter = ',')]
+        validators: Vec<String>,
+    },
+
+    /// Force new era in staking world. Requires sudo.
+    ForceNewEra,
+
+    /// Declare the desire to nominate target account
+    Nominate {
+        #[clap(long)]
+        nominee: String,
+    },
+
+    /// Associate the node with a specific staking account.
+    PrepareKeys,
+
+    /// Call rotate_keys() RPC call and prints them to stdout
+    RotateKeys,
+
+    /// Sets given keys for origin controller
+    SetKeys {
+        /// 64 byte hex encoded string in form 0xaabbcc..
+        /// where aabbcc...  must be exactly 128 characters long
+        #[clap(long)]
+        new_keys: String,
+    },
+
+    /// Command to convert given seed to SS58 Account id
+    SeedToSS58,
+
+    /// Sets lower bound for nominator and validator. Requires root account.
+    SetStakingLimits {
+        /// Nominator lower bound
+        #[clap(long)]
+        minimal_nominator_stake: u64,
+
+        /// Validator lower bound
+        #[clap(long)]
+        minimal_validator_stake: u64,
+
+        /// Maximum number of nominators
+        #[clap(long)]
+        max_nominators_count: Option<u32>,
+
+        /// Maximum number of validators
+        #[clap(long)]
+        max_validators_count: Option<u32>,
+    },
+
+    /// Transfer funds via balances pallet
+    Transfer {
+        /// Number of tokens to send,
+        #[clap(long)]
+        amount_in_tokens: u64,
+
+        /// SS58 id of target account
+        #[clap(long)]
+        to_account: String,
+    },
+
+    /// Send new runtime (requires sudo account)
+    UpdateRuntime {
+        #[clap(long)]
+        /// Path to WASM file with runtime
+        runtime: String,
+    },
+
+    /// Call staking validate call for a given controller
+    Validate {
+        /// Validator commission percentage
+        #[clap(long)]
+        commission_percentage: u8,
+    },
+
+    /// Update vesting for the calling account.
+    Vest,
+
+    /// Update vesting on behalf of the given account.
+    VestOther {
+        /// Account seed for which vesting should be performed.
+        #[clap(long)]
+        vesting_account: String,
+    },
+
+    /// Transfer funds via balances pallet
+    VestedTransfer {
+        /// Number of tokens to send.
+        #[clap(long)]
+        amount_in_tokens: u64,
+
+        /// Seed of the target account.
+        #[clap(long)]
+        to_account: String,
+
+        /// How much balance (in rappens, not in tokens) should be unlocked per block.
+        #[clap(long)]
+        per_block: Balance,
+
+        /// Block number when unlocking should start.
+        #[clap(long)]
+        starting_block: BlockNumber,
+    },
+
+    /// Print debug info of storage
+    DebugStorage,
+
+    /// Deploys a new contract, returns its code hash and the AccountId of the instance
```

Comment:
should a command return something?

Reply:
oh, okay, I see now - please be consistent among all new commands: either every should declare what it prints or none

Reply:
done